### DISCUSSION TITLE
Making an image transparent leaves a little bit left over

### DISCUSSION THREAD

**Discussion by ghost** [dlemstra/Magick.NET#1319] [Help]
Hi, I am new to ImageMagick in general. Please excuse any incorrect usage.
I have attached a copy of the image and the result of me trying to remove it's background, which is 0, 0, 255.
I have tried to use MagickImage.Transparent and MagickImage.TransparentChrome to no avail, as a faint outline is seen around the contents. It is quite faint but I fear that in production it will cause further issues.

I get similar results using bitmap.MakeTransparent.
For ColorFuzz, I have used roughly 40% as it is the maximum before my image starts to distort. Even if I do something like 50%, the outline is strangely still visible. Despite me just talking about this, I have tried a couple other things I've seen that wield the same result. Here is a copy of my code: 
```
image.ColorFuzz = new Percentage(42);
image.Transparent(color);
```

Thank you.

(Also, if it was visible, sorry about opening the same discussion twice, for some reason this discussion was not visible from my other account so I just logged into my old one.)
[Images.zip](https://github.com/dlemstra/Magick.NET/files/10251874/Images.zip)

**Comment by dlemstra** [comment]
I am not an expert on this I would advise you to ask your question here instead: https://github.com/imagemagick/imagemagick/discussions and then come back here to change the command line example to code if you need more help with that.
